src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -103,6 +105,11 @@ public sealed partial class Server : IDisposable {
             _initParams = @params;
             _log = _services.GetService<ILogger>();
 
+            Root = _initParams?.rootUri != null ? _initParams.rootUri.ToAbsolutePath() : _initParams?.rootPath;
+            if (Root != null) {
+                Root = PathUtils.NormalizePathAndTrim(Root);
+            }
+
```

Comment:
Up to you I guess it is `_initParams?.rootUri?.ToAbsolutePath() ?? _initParams?.rootPath`